configure

```diff
@@ -67,6 +67,13 @@ save_configuration
 
 freshen_chef_repo
 
-cd chef_repo
+pushd chef_repo
 wd="$(pwd)"
-exec cinc-solo -c "${wd}/.chef/solo.rb" -E "$CHEF_ENVIRONMENT" -j "${wd}/${CHEF_SOLO_FILE}"
+cinc-solo -c "${wd}/.chef/solo.rb" -E "$CHEF_ENVIRONMENT" -j "${wd}/${CHEF_SOLO_FILE}"
```

Comment:
Be honest, did you strip the `exec` the first time or only after it didn't work?
😀  I usually forget.

My original PR adding these scripts mentions why I prefer to `exec` at the end of a "staging" or "wrapper" script: When running utilities like this I prefer not to leave behind the dangling shell. If this script were to be run as part of a supervised process I'd prefer the process under supervision to be `cinc-solo` itself rather than the `bash` or `sh` process calling it.

Since we're now performing cleanup operations that isn't viable and my preference is overridden.

Reply:
Of course I strip it when it didn't work :laughing: 
Yeah, I understood the reasoning behind using `exec` there and I agree with the principle.